local-volume/provisioner/pkg/common/common.go

```diff
@@ -232,6 +234,9 @@ func ConfigMapDataToVolumeConfig(data map[string]string, provisionerConfig *Prov
 				return fmt.Errorf("Invalid empty block cleaner command for class %v", class)
 			}
 		}
+		if config.MountDir == "" {
+			config.MountDir = ProvisionerDefaultMountRoot
```

Comment:
Do we need to avoid collisions on mountDir? We could generate it from hostDir, like the bootstrapper did. https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/bootstrapper/bootstrapper.go#L89

Reply:
@ianchakeres Frankly I cannot see a value doing it, because having all mounts, even for different classes in the same folder should not create a problem, also it is visible only to the provisioner so I am not sure if it is worth it. Do you see any possible issues?

Reply:
I think we need to fail if MountDir is not specified.  Because not only do you need to specify MountDir in the configmap, you also need to create the hostpath volume for MountDir in the provisioner DaemonSet.